src/implementations/wallet/create.ts

```diff
@@ -36,6 +36,19 @@ export const create = async ({
       );
     }
   }
+  else if (fund === "apothem") {
+    const response = await fetch(`https://faucet.openattestation.com/donate/${wallet.address}`).then((res) =>
```

Comment:
OpenAttestation faucet doesn't support Apothem, does it?

Reply:
XDC Faucet link updated here :- https://faucet.apothem.network